packages/camera/example/lib/main.dart

```diff
@@ -37,81 +44,147 @@ class _CameraExampleHomeState extends State<CameraExampleHome> {
 
   @override
   Widget build(BuildContext context) {
-    final List<Widget> headerChildren = <Widget>[];
-
+    final List<Widget> controlsChildren = <Widget>[];
```

Comment:
I think we should clean up a bit this code to make it easier for people that want to copy the example to start the integration in their code. I think I can work on this.

Reply:
Yes I could work on the initial bit. Would it be good to have comments on this page ?

Reply:
I agree this requires a good refactoring. From someone that as good knowledge of dart for example :p